File: Python-Basics/lunch_break.py
from math import ceil

serial_name = input()
episode_duration = int(input())
break_time = int(input())

time_for_lunch = break_time // 1/8
time_for_chilling = break_time // 1/4
time_left = break_time - time_for_lunch - time_for_chilling
diff = abs(time_left - episode_duration)

if time_left <= episode_duration:
    print(f''' You don't have enough time to watch {serial_name}, you need {ceil(diff)} more minutes.''')
else:
    print( f'You have enough time to watch {serial_name} and left with {ceil(diff)} minutes free time.')

File: JS-Basics/Conditional Statements Exercise/lunch-break.js
function chill(input){
    let serialName = input[0];
    let episodeDuration = Number(input[1]);
    let breakDuration = Number(input[2]);

    let lunchTime = breakDuration * 1/8;
    let chillTime = breakDuration * 1/4;

    let totalTime = breakDuration - lunchTime - chillTime;
    let diff = Math.ceil(Math.abs(totalTime - episodeDuration))
    if(totalTime >= episodeDuration){
        console.log(`You have enough time to watch ${serialName} and left with ${diff} minutes free time.`)
    }
    else{
        console.log(`You don't have enough time to watch ${serialName}, you need ${diff} more minutes.`)
    }
}
chill(["Teen Wolf",
"48",
"60"])
;